.github/workflows/check-layer.yml

```diff
@@ -0,0 +1,18 @@
+name: Yocto Check Layer
+
+on:
+  pull_request:
+
+jobs:
+  yocto-check-layer:
+    runs-on: [self-hosted, x86]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run yocto-check-layer
+        run: |
+          mkdir -p ../build
+          cd ../build
+          kas shell ../meta-qcom-hwe/ci/qcs6490-rb3gen2-core-kit.yml --command "yocto-check-layer-wrapper `pwd`/../meta-qcom-hwe --dependency `pwd`/poky/meta `pwd`/poky/meta-poky `pwd`/meta-qcom --no-auto-dependency"
```

Comment:
The meta-poky is not a dependency of the BSP layer and is there mostly to provide the distro for CI testing